### ISSUE TITLE
Fixes

### ISSUE THREAD

**Issue by melinashak-noaa** [nwfsc-fram/boatnet#2315]
- [ ] Get ride of side arrows on headings
- [ ] Search not working 
- [ ] WM 8 & 15 Missing from drop down and data
- [ ] Ratio for WM 15, add to model as well
- [ ] Adjust settings for WCGOP/ASHOP, move toggle

**Comment by phillipbizzell-NOAA** [comment]
Couple more things I forgot to mention yesterday not sure how you want to curate these items
Toggling codes on drops a number of fields including: 
1. TRIPS - Dept and Ret port (should not be coded), Some values for Fishery including LE Sablefish
3. HAULS - OTC WT Method, Some Gear Types (Longline fixed)
4. CATCH -  Weight Method Visual, Top name "Miscellaneous" 

Editing - 
When editing a field the view resets to the left, can it be static?

**Comment by melinashak-noaa** [comment]
Got it, working on this, for the species, is there a list that maps species codes to species names?

**Comment by phillipbizzell-NOAA** [comment]
Hesitating on this one, because we're generally not familiar with all of the species codes.  Would probably be better to turn on a Code column or hyphenate Code - Species when toggled.  I'll ask around about this..

**Comment by melinashak-noaa** [comment]
I see, if it's really not needed I can leave out species codes all together and just show the catch category codes. Some of the species codes are missing from our database anyways so it would take some time to update all the database records.

**Comment by phillipbizzell-NOAA** [comment]
Yeah, you could leave the species comp level codes out for now if it's additional work (we may want to add a code column later on at some point).  Personally I'd love to see the catch level species toggled to the catch codes.  We're doing away with the "catch category" sampling structure, but the catch codes are useful abbreviations and observers could continue to use these, especially if we ever move forward with the Fixed Gear Tally app. Any item at the catch level will have a corresponding 3-5 letter code.

**Comment by Jeibner-NOAA** [comment]
Not sure if this is the appropriate ticket for this comment, or if it was mentioned elsewhere, but the "Logbook Type" field on the Trip screen is only a code at the moment.  Could we get the logbook names displayed instead?  Unfortunately it will  take up more real estate, but we've never used the codes for Logbook Type, so they aren't familiar.

**Comment by melinashak-noaa** [comment]
okay got the catch numeric code implemented now. And @Jeibner-NOAA yes I'll work on getting the logbook type field into the data. So then it should display something like 'WOC Trawl' instead of 1

**Comment by Jeibner-NOAA** [comment]
Thanks, @melinashak-noaa , that would be great.

**Comment by phillipbizzell-NOAA** [comment]
This looks good Melina.  One thing I'm noticing is that PHLB sometimes has a species code at the catch level. I would expect to see "PHLB" at the upper level.  101 is displayed when 1 sample method is used for lengths, populating the specimen cnt at the top level. Probably want to make all records consistent. You could put the specimen count on the upper level - the counts are split by BS method, but that information is not displayed on this screen, so they could be combined.  Otherwise make all counts display in the drop down. Note that the discard reason 16 lives at the catch level and Biospecimen level for some reason, not sure why it's only displayed in the drop down?!
![PHLB Capture](https://user-images.githubusercontent.com/48926038/119045707-aaebfc00-b970-11eb-8afc-8244dabeaa97.PNG)

**Comment by melinashak-noaa** [comment]
Got it, fixed the issue so now PHLB is displayed the top level and the children '101 - pacific halibut' underneath. There was some weird logic combining the parent and child so got rid of that. And for discard reason, we'll be updating our data model so once that's done I'll update discard reason to show up in the upper level, not the child level.
